core/src/test/java/org/elasticsearch/search/fetch/subphase/highlight/HighlighterSearchIT.java

```diff
@@ -2435,6 +2435,7 @@ public void testPostingsHighlighterCommonTermsQuery() throws IOException {
 
     private static XContentBuilder type1PostingsffsetsMapping() throws IOException {
         return XContentFactory.jsonBuilder().startObject().startObject("type1")
+                .startObject("_all").field("enabled", true).endObject()
                 .startObject("properties")
                 .startObject("field1").field("type", "text").field("index_options", "offsets").endObject()
                 .startObject("field2").field("type", "text").field("index_options", "offsets").endObject()
```

Comment:
Ok thanks for the explanation. BTW are we planning to add support for `all_fields` in the multi match query ?